components/sidebar/sidebar_next_steps/sidebar_next_steps.scss

```diff
@@ -1,8 +1,4 @@
 .SidebarNextSteps {
```

Comment:
Looks like there's a number of `.NextStepsView__nextStepsCards` styles that can all be removed.